pkg/terminal/command.go

```diff
@@ -1132,6 +1135,56 @@ func tracepoint(t *Term, ctx callContext, args string) error {
 	return setBreakpoint(t, ctx, true, args)
 }
 
+func getLocation(t *Term, ctx callContext) (*api.Location, error) {
+	if ctx.scoped() {
+		locs, err := t.client.Stacktrace(ctx.Scope.GoroutineID, ctx.Scope.Frame, false, nil)
+		if err != nil {
+			return nil, err
+		}
+		if ctx.Scope.Frame >= len(locs) {
+			return nil, fmt.Errorf("Frame %d does not exist in goroutine %d", ctx.Scope.Frame, ctx.Scope.GoroutineID)
+		}
+		return &locs[ctx.Scope.Frame].Location, nil
+	} else {
+		state, err := t.client.GetState()
+		if err != nil {
+			return nil, err
+		}
+		if state.SelectedGoroutine != nil {
+			return &state.SelectedGoroutine.CurrentLoc, nil
+		} else {
+			thread := state.CurrentThread
+			loc := api.Location {
+				PC: thread.PC,
+				File: thread.File,
+				Line: thread.Line,
+				Function: thread.Function,
+			}
+			return &loc, nil
+		}
+	}
+}
+
+func edit(t *Term, ctx callContext, args string) error {
+	loc, err := getLocation(t, ctx)
+	if err != nil { return err }
+
+	var editor string
+
+	editor = os.Getenv("DELVE_EDITOR")
```

Comment:
nit: Just use an `:=` here instead of declaring the variable above.

Reply:
Either that or declare and cleanup below with something like:

```
var editor string
if editor = os.Getenv("DELVE_EDITOR"); editor == "" {
        if editor = os.Getenv("EDITOR"); editor == "" {
                return <err>
        }
}
```

Reply:
Yes good point. I started with a := but had another one in the if block which wasn't right and got confused. The declare and cleanup thing looks quite Gothonic (?) I'll have to remember that one :)